vendor.conf

```diff
@@ -120,7 +120,7 @@ github.com/googleapis/gax-go v2.0.0
 google.golang.org/genproto 694d95ba50e67b2e363f3483057db5d4910c18f9
 
 # containerd
-github.com/containerd/containerd ceba56893a76f22cf0126c46d835c80fb3833408
+github.com/containerd/containerd 894b81a4b802e4eb2a91d1ce216b8817763c29fb # v1.2.6
```

Comment:
Ah, I know the problem; the _vendored_ version of containerd (which has the containerd client code) is currently using a version from master. This is sometimes needed to make use of new features in the client that are not yet in a release.

You can drop this commit, as it would revert the containerd client to an older version; after doing so, things should work 👍

Reply:
Thanks for your point. :+1:  I will update it.